bin/linting.mjs

```diff
@@ -0,0 +1,135 @@
+import { dirname, resolve } from "node:path";
+import { readFile, writeFile } from "node:fs/promises";
+import { pkgUp } from "pkg-up";
+import { parseNi, run as runNi } from "@antfu/ni";
+// ni uses this so we do too
+import prompts from "@posva/prompts";
+
+const TEMPLATE_HEADER = '/* eslint-disable */';
+
+const TEMPLATE_CJS = `${TEMPLATE_HEADER}
+
+/**
+ * @type {import('eslint').Linter.FlatConfig[]}
+ */
+module.exports = [
+{{configs}}
+];
+`;
+
+const TEMPLATE_ESM = `${TEMPLATE_HEADER}
+
+{{imports}}
+
+/**
+ * @type {import('eslint').Linter.FlatConfig[]}
+ */
+const config = [
+{{configs}}
+];
+ 
+export default config;
+`;
+
+
+
+const ni = (args) => {
+  return runNi(parseNi, args);
+}
+
+const confirm = async (message) => {
+  return (await prompts({type: 'confirm', name: 'confirm', message: message, initial: true })).confirm;
+}
+
+const createConfig = (pkg) => {
+  const isEsm = pkg.type === 'module';
+  console.log('Format (based on package.json -> type):', isEsm ? 'ES Module' : 'CommonJS');
+  const dependencies = pkg.dependencies ?? {};
+  const template = isEsm ? TEMPLATE_ESM : TEMPLATE_CJS;
+  const configs = isEsm
+    ? ['...recommended']
+    : [`...require('@atmina/linting/eslint/recommended')`];
+  const imports = isEsm
+    ? [`import recommended from '@atmina/linting/eslint/recommended.js'`]
+    : [];
+
+  if (dependencies['tailwindcss']) {
+    console.log('+ Tailwind CSS');
+    if (isEsm) {
+      imports.push(`import tailwind from '@atmina/linting/eslint/tailwind.js'`);
+      configs.push('tailwind');
+    } else {
+      configs.push(`require('@atmina/linting/eslint/tailwind')`);
+    }
+  }
+
+  if (dependencies['react'] || dependencies['next']) {
```

Comment:
The next config already includes react, so I don't know if there is any benefit (or drawback) to importing it additionally. Otherwise, this should probably be `dependencies['react'] && !dependencies['next']`